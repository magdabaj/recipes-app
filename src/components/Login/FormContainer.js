// eslint-disable-next-line
import React from 'react';
import styled from 'styled-components';

const FormContainer = styled.div`
  display: flex;
  align-items: center;
  // flex-direction: row;
  // justify-content: center;
  // align-content: space-evenly;
  padding: 20px;

  @media (max-width: 768px) {
    width: 100%;
    padding: 0;
  }
`;

export default FormContainer;
