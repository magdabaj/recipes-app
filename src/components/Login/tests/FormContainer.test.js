import FormContainer from "../FormContainer";
import renderer from 'react-test-renderer'
import React from "react";

test('FormContainer', () => {
    const tree = renderer.create(<FormContainer/>).toJSON()
    expect(tree).toMatchSnapshot()
})