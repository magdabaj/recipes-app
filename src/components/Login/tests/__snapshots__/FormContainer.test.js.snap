// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormContainer 1`] = `
<div
  className="FormContainer-sc-1xox95j-0 fxVHqF"
/>
`;
